src/Logger/VWOLogger.php

```diff
@@ -86,15 +86,11 @@ private function getFormatter()
         return new LineFormatter("[%datetime%] %channel%.%level_name%: %message%\n");
     }
 
-    /***
-     * to add logs to monolog
-     *
-     * @param  $msg
-     * @param  int $level
-     * @return mixed|void
+    /**
+     * @inheritdoc
      */
-    public function log($msg, $level = Logger::INFO)
+    public function log($level, $message, array $context = array())
```

Comment:
we can omit declaring type in parameters(for third parameter), this might not work in lower version of PHP
like log($level, $message, $context = array()) Or log($level, $message, $context = [])